modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -430,6 +431,30 @@ object SwaggerUtil {
       .map(_.toMap)
   }
 
+  def copyOperation(operation: Operation): Operation =
+    new Operation()
```

Comment:
No concerns that this is shallow?

Reply:
Ah, I see that it's currently only used for one thing. Hopefully this doesn't bite us.

Reply:
We generally treat openapi model objects as immutable (with this new exception), so it's probably not an issue.  I added the copying bit out of an overabundance of caution that probably isn't even needed, but I feel safer that way.